libr/core/cmd_print.c

```diff
@@ -3132,11 +3137,11 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 	ut64 off = core->offset;
 	ut64 from = UT64_MAX;
 	ut64 to = 0;
-	RTable *t = r_core_table (core);
-	t->showSum = true;
-	RList *list = r_core_get_boundaries_prot (core, -1, NULL, "search");
+
+	list = r_core_get_boundaries_prot (core, -1, NULL, "search");
 	if (!list) {
-		return 1;
```

Comment:
Is result having more values than 0 and 1? Because maybe we can boolify it

Reply:
Doesn't seem like exact value is used anywhere. In the success case it returns len - number of blocks. But len gets calculated only for json which is the case where count is least useful so most likely the result doesn't get properly used at all and it can be boolified.

Reply:
imho in case of confusion, better go for bool

Reply:
Done.